app/styles/components/_session-offering.scss

```diff
@@ -0,0 +1,7 @@
+.plus-button, .minus-button {
```

Comment:
Plus / minus button style should go in its own scss file so we can move it into a pod later.